app/lib/__tests__/cloneutils-test.js

```diff
@@ -0,0 +1,74 @@
+jest.unmock("../cloneutils.js")
+
+import { clone } from "../cloneutils.js"
+import rmdir from "rimraf"
+import sinon from "sinon"
+
+const fs = require("fs")
+
+const TEST_REPO = "https://github.com/education/classroom-desktop"
+const TEST_FAKE_REPO = "https://github.com/education/a-repo-that-will-never-exist"
```

Comment:
I've never seen anyone define constants in camelCase, mostly uppercase. Seems to be the convention in most languages :)